src/inspectornetworkwatcher.h

```diff
@@ -0,0 +1,22 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef INSPECTORNETWORKWATCHER_H
+#define INSPECTORNETWORKWATCHER_H
+
+#include "networkwatcherimpl.h"
+
+class InspectorNetworkWatcher final : public NetworkWatcherImpl {
```

Comment:
Consider if names like InactiveNetworkWatcher or OfflineNetworkWatcher are better, instead of coupling it to use by Inspector.